modules/contentbox/models/content/ContentService.cfc

```diff
@@ -127,10 +126,7 @@ component extends="cborm.models.VirtualEntityService" singleton {
 	 * @async Run it asynchronously or not, defaults to false
 	 */
 	ContentService function clearPageWrapper( required any slug, boolean async = false ){
-		variables.cacheBox
-			.getCache( variables.settingService.getSetting( "cb_content_cacheName" ) )
-			.clear( "cb-content-wrapper-#cgi.HTTP_HOST#-#arguments.slug#/" );
-		return this;
+		return clearPageWrapperCaches( argumumentCollection = arguments );
```

Comment:
Before you ask about this :) - Since we are accounting for the full RC now, there is no way to accurately clear the single page wrapper so we are just clearing all cache items by snippet for the page now.